[#3676] Fix regression issue preventing file browser refresh.

Maurier opened the pull request:
https://github.com/hydroshare/hydroshare/issues/3676 Fixes a regression issue preventing file browser to refresh automatically after an aggregation is created.

<!--

Please read, and add your text at the bottom of this message.

Thanks for contributing code to HydroShare. In order to maintain code quality and expedite this process, please assist the development team by making sure the following is present in this pull request.

For more information, see https://docs.google.com/document/d/1dzxqlZW5fKNEyQSeKiSFq-SmS-VOPCva95XXkBjPExs

-->

### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

<!-- Enter steps that a QA engineer, stakeholder, or user documentation writer would follow to test the positive or "successful" case of the functionality your code provides or fixes -->

- [ ] Automated Testing

<!-- Our Jenkins Instance is set up to automatically test every commit from a pull request. Code coverage must not decrease so new functionality or code paths added during a bug fix must have appropriate tests written. Every test must pass, including PEP8 code formatting tests. -->

- [ ] Sufficient User and Developer Documentation

<!-- Please email your positive test case lbrazil@cuahsi.org, who will make the decision regarding user documentation. -->

- [ ] Passing Jenkins Build

<!-- Our Jenkins Instance is set up to automatically test every commit from a pull request. Every test must pass, including PEP8 code formatting tests. -->

- [ ] Peer Code review and approval

<!-- This is the process by which a peer developer on the HydroShare team will read the changeset, provide feedback, and ultimately give a formal approval to the code before it passes PR status. -->

### Positive Test Case
1. Create a resource.
2. Create an aggregation on this resource.
3. Undo this aggregation by right-clicking on the file and selecting "Remove content type".
4. Restore the file's content type by right-clicking it and selecting an option under "Setcontent type".
5. At this point verify that your changes were applied automatically without the need to refresh the page.

Maurier commented:
@pkdash Thanks, I can confirm this works as intended. I tested it extensively.

Maurier commented:
@sblack-usu, @dtarb Let me know if this merits a patch to www

dtarb commented:
@Maurier given that the next release is imminent this is probably best just including in the next release.  But I am a bit unclear how this impacts users.  If it is hampering users, and the next release will drag on, it may warrant a hotfix.  I suggest discussing on the Monday call.

Maurier commented:
This fix addresses issues such as https://github.com/hydroshare/hydroshare/issues/3671
In short, the browser was not automatically reloaded after adding metadata to a file, so a user would not see the metadata form on the right panel until the page is refreshed. These changes fix that so that the metadata form is displayed without refreshing the page.

@pkdash should be able to weight in in the call since he's the one who brought this to my attention.

dtarb commented:
Thanks.  I had not connected this with #3671, but it makes sense now and am glad it is getting fixed.
